integration-tests/src/test/java/oracle/kubernetes/operator/utils/Operator.java

```diff
@@ -501,6 +512,49 @@ private void initialize(
     }
   }
 
+  /**
+   * restart operator pod using replicas
+   *
+   * @throws Exception
+   */
+  public void restartUsingReplicas() throws Exception {
+    stopUsingReplicas();
+    startUsingReplicas();
+  }
+
+  /**
+   * stop operator pod by scaling its replicaset to 0
+   *
+   * @throws Exception
+   */
+  public void stopUsingReplicas() throws Exception {
+    String cmd = "kubectl scale --replicas=0 deployment/weblogic-operator" + " -n " + operatorNS;
+    logger.info("Undeploy Operator using command:\n" + cmd);
+
+    ExecResult result = ExecCommand.exec(cmd);
+
+    logger.info("stdout : \n" + result.stdout());
+
+    logger.info("Checking if operator pod is deleted");
+    verifyPodDeleted();
+  }
+
+  /**
+   * start operator pod by scaling its replicaset to 1
+   *
+   * @throws Exception
+   */
+  public void startUsingReplicas() throws Exception {
+    String cmd = "kubectl scale --replicas=1 deployment/weblogic-operator" + " -n " + operatorNS;
+    logger.info("Deploy Operator using command:\n" + cmd);
+
+    ExecResult result = ExecCommand.exec(cmd);
```

Comment:
Use TestUtils.exec(cmd)

Reply:
For README.txt, I don't know why my changes are not showing up here. I checked both ,my local and the link below, both are correct and usecases for server discovery are erased

https://github.com/oracle/weblogic-kubernetes-operator/blob/server-discovery-tests/integration-tests/README.md

Reply:
The link you sent doesn't have correct content. It still contains use cases. Compare with the one in develop branch - https://github.com/oracle/weblogic-kubernetes-operator/tree/develop/integration-tests

Reply:
I am not sure why you still seeing the old one.  Changes in READMA.txt were from these two commits:

https://github.com/oracle/weblogic-kubernetes-operator/pull/993/commits/c8a570ec4f6f6a758c19845929187eca6072b57b
https://github.com/oracle/weblogic-kubernetes-operator/pull/993/commits/9fcbf2df07896713cbb11365bac29f524b467c3b


Also from the link https://github.com/oracle/weblogic-kubernetes-operator/blob/server-discovery-tests/integration-tests/README.md I sent above, I searched "Server Discovery" and got nothinbg, what I see is:

Session Migration | Use Case
-- | --
Primary server repick | a backup server becomes the primary server when a primary server fails
HTTP session nigration | verify in-memory HTTP session State replication

Use Cases covered in integration tests for the operator is available here
Directory Configuration and Structure

Above matched my local

Reply:
double checked 

https://github.com/oracle/weblogic-kubernetes-operator/pull/993/files

Changes are correctly there as well